src/Symfony/Bridge/Twig/Tests/Node/TransNodeTest.php

```diff
@@ -50,6 +51,10 @@ protected function getVariableGetterWithoutStrictCheck($name)
 
     protected function getVariableGetterWithStrictCheck($name)
     {
+        if (class_exists(LoopIterator::class)) {
+            return \sprintf('(array_key_exists("%1$s", $context) ? $context["%1$s"] : throw new RuntimeError(\'Variable "%1$s" does not exist.\', 0, $this->source))', $name);
```

Comment:
see twigphp/Twig#3933 and twigphp/Twig#3936

Reply:
I suggest adding a comment to make it clear that the code path below is for Twig 3.x and older (allowing to clean this method when dropping support for Twig 3)

Reply:
I added one